zerver/management/commands/generate_realm_creation_link.py

```diff
@@ -16,10 +18,18 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         # type: (*Any, **Any) -> None
-        url = generate_realm_creation_url()
-        self.stdout.write(
-            "\033[1;92mPlease visit the following secure single-use link to register your ")
-        self.stdout.write("new Zulip organization:\033[0m")
-        self.stdout.write("")
-        self.stdout.write("    \033[1;92m%s\033[0m" % (url,))
-        self.stdout.write("")
+
+        try:
+            # first check if the db has been initalized
+            Realm.objects.first()
+
+            url = generate_realm_creation_url()
+            self.stdout.write(
+                "\033[1;92mPlease visit the following secure single-use link to register your ")
+            self.stdout.write("new Zulip organization:\033[0m")
+            self.stdout.write("")
+            self.stdout.write("    \033[1;92m%s\033[0m" % (url,))
+            self.stdout.write("")
+
+        except ProgrammingError:
+            print("The Zulip database does not appear to exist. Have you run initialize-database?")
```

Comment:
I would probably just add a sys.exit(1) in the `except:` handler and just have the `Realm.objects.all()` be in the `try:` block.

Otherwise this looks good!